#Header header

string text
string language
float32 volume
float32 pitch
float32 rate
